charts/kube-prometheus-stack/templates/_helpers.tpl

```diff
@@ -96,9 +96,18 @@ Allow the release namespace to be overridden for multi-namespace deployments in
   {{- end -}}
 {{- end -}}
 
+{{/* Allow KubeVersion to be overridden. */}}
+{{- define "kube-prometheus-stack.ingress.kubeVersion" -}}
+  {{- if .Values.kubeVersionOverride -}}
+    {{- .Values.kubeVersionOverride -}}
+  {{- else -}}
+    {{- print .Capabilities.KubeVersion.Version -}}
+  {{- end -}}
```

Comment:
```suggestion
  {{- print  .Values.kubeVersionOverride | default .Capabilities.KubeVersion.Version -}}
```

Reply:
@JoshVee that is further simplified as we established earlier that all templates come back as strings
```
{{- default .Capabilities.KubeVersion.Version .Values.kubeVersionOverride -}}
```

Reply:
Sounds good. Thanks.

Thanks for reviewing it. Would be great to have it merged if possible.